torch/_jit_internal.py

```diff
@@ -740,6 +741,11 @@ def decorator(fn):
     return decorator
 
 
+def _drop(fn):
```

Comment:
nit: could you add some extra docs here? e.g. why we need it (classes try to compile all methods by default, and unused/ignore don't work because they have return types), use case (non-method classes only)